src/lib/WIFI/devWIFI.cpp

```diff
@@ -85,6 +85,15 @@ static bool target_complete = false;
 static bool force_update = false;
 static uint32_t totalSize;
 
+void setWifiUpdateMode()
+{
+  // No need to ExitBindingMode(), the radio will be stopped stopped when start the Wifi service.
+  // Need to change this before the mode change event so the LED is updated
+  InBindingMode = false;
+  connectionState = wifiUpdate;
+  devicesTriggerEvent();
```

Comment:
`devicesTriggerEvent` isn't needed, since if we're not in wifi update mode then there will be an event triggered, and if we're already in wifi update mode then there is no need to trigger the event.

Reply:
Thats true now.  But if I remove the `devicesTriggerEvent` from here I need it below for the case where we're in the error state and flip to wifi mode.

On writing this I think in that case we should continue with the error state on the LEDs.